cmd/goat/blob.go

```diff
@@ -28,6 +28,11 @@ var cmdBlob = &cli.Command{
 					Aliases: []string{"o"},
 					Usage:   "directory to store blobs in",
 				},
+				&cli.StringFlag{
+					Name:    "pds-host",
+					Usage:   "URL of the PDS to export blobs from (overrides DID doc)",
+					EnvVars: []string{"ATP_PDS_HOST"},
```

Comment:
nit pick, but this could collide with an env var with the same name used for login, in a confusing way. basically we use `ATP_PDS_HOST` to hint login/auth. I think we should use `ATP_BLOB_HOST` here, and `blob-host` as the arg name. Or maybe `ATP_PDS_BLOB_HOST`, but I like the shorter `ATP_BLOB_HOST` in the public API.

Reply:
also, I guess if a DID was provided (for the account), and `ATP_BLOB_HOST` was provided, we could just skip identity resolution? that could help with situations where the identity is broken for whatever reason.

Reply:
Yeah, given that, maybe the easiest thing to do would be to just remove the env va?
```suggestion
```